lib/galaxy/tool_util/xsd/galaxy.xsd

```diff
@@ -1788,7 +1788,7 @@ statement. A good example tool that demonstrates many conditional parameters is
     </param>
     <when value="tsv">
         <param name="input_table" type="data" format="tabular" label="Tabular File" argument="--input-fp"/>
-        <param name="process_obs_metadata" type="select" label="Process metadata associated with observations when converting" argument="--process-obs-metadata">
+        <param type="select" label="Process metadata associated with observations when converting" argument="--process-obs-metadata">
```

Comment:
```suggestion
        <param argument="--process-obs-metadata" type="select" label="Process metadata associated with observations when converting">
```
Since this replaces the `name` argument I think it should come first.
Also we should update https://galaxy-iuc-standards.readthedocs.io/en/latest/best_practices/tool_xml.html#parameter-name-argument-and-help once merged.

Reply:
update to help started: https://github.com/galaxy-iuc/standards/pull/55